### PR TITLE
Directs intermediate and final pdf output of latexmk to build directory

### PR DESCRIPTION
**Pull request by dlurton**
This helps to keep the directories containing our `.tex` files much cleaner.

This PR builds on the changes from #20, which changes the directory structure a bit.  Ideally, #20 is merged first.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: psr/PSR-TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Instructions
+
+- Copy this file to `proposals/<unique-feature-name>.md`.
+- Fill in the sections below.
+- Delete this section.
+- Submit a pull request your PSR.
+
+# PartiQL Specification Request
```
Comment (almann) [inline psr/PSR-TEMPLATE.md:8]: Is this part of the change?  Seems unrelated...

**Review by almann — COMMENTED** [review]
Did you mean to put the PSR stuff as part of this PR?

**Comment by dlurton** [comment]
Oops, no.  Just the other stuff. I thought I had fixed that.

**Comment by dlurton** [comment]
To be clear, only this commit was intended to bed in this PR. I will fix tomorrow.

https://github.com/partiql/partiql-spec/commit/76320926ebea683f9f0fb99bb554ecc794759a5e

**Comment by almann** [comment]
> To be clear, only this commit was intended to bed in this PR. I will fix tomorrow.
> 
> [7632092](https://github.com/partiql/partiql-spec/commit/76320926ebea683f9f0fb99bb554ecc794759a5e)

Thought so--once you rebase, I will happily :ship:.

**Comment by jpschorr** [comment]
Obsoleted by #30
